Traceback (most recent call last):
  File "updria.py", line 87, in <module>
    main()
  File "updria.py", line 81, in main
    res = updria(opts, ts)
  File "/home/gredondi/src/updria/_updria.py", line 1131, in updria
    recblockres = measure('rec_block_time', recblock, paramts, abstract_predicates_dict,\
  File "/home/gredondi/src/updria/_updria.py", line 108, in measure
    return func(*args, **kwds)
  File "/home/gredondi/src/updria/_updria.py", line 1003, in recblock
    assert str(s.check) == 'unknown'
AssertionError
real 7202.22
user 7200.83
sys 1.09
